.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  extends: [require.resolve('@backstage/cli/config/eslint')],
```

Comment:
I would be inclined excluding this dependency if at all possible. I dont think it is worth the weight if just for eslint config.

Reply:
All unneeded dependencies are to be excluded